src/routes/transactionWrapper.js

```diff
@@ -1,9 +1,22 @@
 import { sequelize } from '../models';
+import handleErrors from '../lib/apiErrorHandler';
+
+const namespace = 'SERVICE:WRAPPER';
+
+const logContext = {
+  namespace,
+};
 
 export default function transactionWrapper(originalFunction) {
   return async function wrapper(req, res, next) {
     return sequelize.transaction(async () => {
-      const result = await originalFunction(req, res, next);
+      let result;
+      try {
+        result = await originalFunction(req, res, next);
+      } catch (err) {
+        await handleErrors(req, res, err, logContext);
+        throw err;
+      }
```

Comment:
We don't want to throw here, since the error handler already takes care of any error and use the passed in `namespace ` in the log.

Reply:
At this point, the transaction should have already been rolled back.

Reply:
If we were to leave it, in case the body of the try/catch throws an unexpected exception, we would crash the server.

Reply:
I think I have adjusted this.